acceptance_tests/features/sensitive_data.feature

```diff
@@ -0,0 +1,7 @@
+Feature: Sample data in the case is sensitive and must be redacted
+
+  Scenario: A case is loaded and sensitive data can be changed
+    Given sample file "sensitive_data_sample.csv" with sensitive column PHONE_NUMBER is loaded successfully
+    When an UPDATE_SAMPLE_SENSITIVE event is received
```

Comment:
Since we're hardcoding it to updated/check the phone number, I think it should mention it in the step e.g. 
`an UPDATE_SAMPLE_SENSITIVE event is received updating the PHONE_NUMBER`